### PR TITLE
Add glif_psc_double_alpha model

### PR DESCRIPTION
**Pull request by shixnya**
Continuation of PR #2861 that was accidentally closed.
The most recent change contains rebasing to the updated upstream/master, and fixing one of the newly added test cases in the updates.

Hopefully, this version passes all of the tests.

### CODE REVIEW

**Comment by heplesser** [comment]
@shixnya @nicolossus No problem to proceed here.
@clinssen @janskaar Could you re-review here quickly?

**Review by nicolossus — APPROVED** [review]
LGTM!

**Review by janskaar — APPROVED** [review]
Everything looks good to me.

**Review by clinssen — APPROVED** [review]

File: pynest/examples/glif_psc_double_alpha_neuron.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Current-based generalized leaky integrate and fire (GLIF) neuron with double alpha
```
Comment (jessica-mitchell) [inline pynest/examples/glif_psc_double_alpha_neuron.py:23]:
```suggestion
Current-based generalized leaky integrate and fire (GLIF) neuron with double alpha \
```

Comment (jessica-mitchell) [reply pynest/examples/glif_psc_double_alpha_neuron.py:23]: The title doesn't render in HTML if it goes over two lines without the `\`

File: pynest/examples/glif_psc_double_alpha_neuron.py [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+synaptic function example
+------------------------------------------------------------------------
+
+Simple example of how to use the ``glif_psc_double_alpha`` neuron model that illustrates
+difference from the ``glif_psc`` neuron model.
```
Comment (jessica-mitchell) [inline pynest/examples/glif_psc_double_alpha_neuron.py:28]:
```suggestion
differences from the ``glif_psc`` neuron model.
```

**Review by jessica-mitchell — CHANGES_REQUESTED** [review]
Thanks @shixnya , I just a have couple of minor changes for the docs

**Review by jessica-mitchell — APPROVED** [review]
thanks!

**Comment by shixnya** [comment]
Thank you so much, all, for your help!

**Pull request merged**